.drone.yml

```diff
@@ -41,7 +41,7 @@ pipeline:
       - cp -R . /tmp/3/ && cd /tmp/3/
       - git submodule update --init --recursive --jobs 7
       - export PATH=/tmp/4/project/scripts:$PATH
```

Comment:
I thought this export would be enough for drone to pick `./project/scripts/sbt` over the default.

Reply:
Yeah I don't know why that doesn't work.

Reply:
Ah, it's the wrong /tmp folder

Reply:
:man_facepalming: